### DISCUSSION TITLE
Release schedules

### DISCUSSION THREAD

**Discussion by summongod** [Blackymas/NSPanel_HA_Blueprint#580] [General]
Hi Folks,

Upgrading my panel is a real pain. I have to load Tasmota to upload the tft file then load the image using the serial pins. I realise these are issues I'm going to have deal with due to esphome handling of the serial interface etc etc. My question is really about frequency of releases. I'd rather only do this 'upgrade' infrequently so having an idea about whether a new release is just around the corner would help me quite a lot.

Do you have any thoughts about this?

I'm really grateful for the work you are doing and the results are fantastic.
S.

**Comment by Blackymas** [comment]
this is all very spontaneous as we do not do it professionally and therefore there is no direct release plan
